.markdownlint.yaml

```diff
@@ -0,0 +1,47 @@
+# markdownlint-cli configuration file (YAML)
+# markdownlint rules: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
```

Comment:
How do you pass the linter to enforce these rules ?

Reply:
I ran it manually, but I just commited a pre-commit hook to enforce it before commits.

https://github.com/System73/edge-analytics-grafana-setup/pull/1/commits/d9c9f507ed4395d6b10324a7573850f5d41bcd2e